styles.scss

```diff
@@ -488,7 +509,7 @@ blockquote {
   @media screen and (max-width: $viewport-sm) {
     margin-top: 190px;
 
-    >h1 {
+    > h1 {
```

Comment:
Out of curiosity, what are you using as a CSS formatter?

Reply:
I think it's Prettier meddling with stuff it shouldn't meddle with, sorry about that I'll revert the formatting changes 😬

Reply:
No it's okay, I like the idea of having a consistent CSS code formatting, you can leave these changes 👍